flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/internals/publisher/fanout/FanOutRecordPublisherConfiguration.java

```diff
@@ -134,6 +134,46 @@
 	 */
 	private final int listStreamConsumersMaxRetries;
 
+	/**
+	 * Max retries for the describe stream operation.
+	 */
+	private final int describeStreamMaxRetries;
```

Comment:
@xiaolong-sn it would make sense to move the `DescribeStream` bits to you config PR since that has not been reviewed yet

Reply:
I'll do that later.